tests/test_jointcal_cfht.py

```diff
@@ -176,6 +176,18 @@ def test_jointcalTask_2_visits_constrainedAstrometry_4sigma_outliers(self):
 
         self._testJointcalTask(2, dist_rms_relative, self.dist_rms_absolute, None, metrics=metrics)
 
+    def test_jointcalTask_2_visits_constrainedAstrometry_astrometryOutlierRelativeTolerance(self):
+        """Test that astrometryOutlierRelativeTolerance changes the fit.
+        A 5% cut here results in the fit ending quite early, with expectedly
```

Comment:
If you're going to use `0.01` in the test below, this docstring needs to say `A 1% cut...`. Also, the sentence itself is less correct now, since the chi2 is not "much worse" (1129->1427).